site/server/app/views/templates/library/index.scala.html

```diff
@@ -1,4 +1,4 @@
-@(library: shared.Library, section: shared.Section)(implicit request: RequestHeader)
+@(library: shared.Library, section: shared.Section, progress: Int = 0)(implicit request: RequestHeader)
```

Comment:
I recall there being a warning by the use of default args values in templates. We should look into it.